index.html

```diff
@@ -90,6 +121,48 @@
 
 	<script>
 
+	$(document).ready(function() {
```

Comment:
Could you please adjust the indentation of the following lines? Actually I really cannot find the end of the `ready` call.